algorithms/closest_pair.py

```diff
@@ -84,7 +84,12 @@ def _distance_between(point_1, point_2):
 
```

Comment:
When you say you can skip the square root, do you mean that without it you can still compare values and be able to find the smallest? If you want to find the _actual_ distance between them you still need the root, right?